packages/app/src/store/studio-store.ts

```diff
@@ -862,7 +867,7 @@ export const useStudioStore = defineStore('studioRecorder', {
     },
 
     needsUrl: (state) => {
-      return state.isActive && !state.url && !state.isFailed
+      return state.isActive && !state.url && !state.isFailed && state._hasStarted
```

Comment:
This was causing the AUT URL to get removed whenever the panel was open at all, even if it was just the new test screen